### ISSUE TITLE
автоматический dpaste для кода

### ISSUE THREAD

**Issue by hulygun** [django-ru/help-bot#3]
Декомпозиция #1 

Вижу 2 пути решения: 
1. Ручной режим (с использованием команды, например `/code`), в таком случае непонятно в каком виде возвращать ссылку с сервиса гиста
2. Автоматический с удалением сообщения. В таком случае боту нужны права на удаление сообщений и как детектить, что сообщение это кусок кода

**Comment by lorddaedra** [comment]
наверное, нужен ручной режим с удалением сообщения и написанием нового со ссылкой на dpaste.de :-)

**Comment by lorddaedra** [comment]
а потом уже можно улучшать)

**Comment by lorddaedra** [comment]
и если не получилось сделать ссылку на dpaste.de, то удалять не надо) то есть удаляем только когда уже есть ссылка)

**Comment by hulygun** [comment]
Если ручной, то нужно как-то детектить то сообщение, которое нужно задепастить, например через репли

**Comment by lorddaedra** [comment]
да, это норм, кто угодно может ответить /code на слишком длинное сообщение с кодом и оно удалится после того, как dpaste.de сделает новый сниппет с кодом и бот вставит в чат ссылку на него

**Comment by lorddaedra** [comment]
если будут злоупотреблять - то можно будет административным путём пресечь это...

**Comment by hulygun** [comment]
По началу много кто будет баловаться

**Comment by lorddaedra** [comment]
ну, это всё довольно быстро решается, если в чате есть модераторы - можно кикнуть из чата или отключить на какое-то время возможность писать в чат
